### DISCUSSION TITLE
runelite hd117

### DISCUSSION THREAD

**Discussion by RodriguezzzL** [runelite/runelite#18286] [Q&A]
Hey guys! so i've been trying to use the 117 plug in and for some reason every time i use it i only get around 30fps My pc has 5800x and a RTX 4070 so i know i can run it but i cant seem to find any setting to fix this issue to get better performance? What can i do to fix this?

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
